### PR TITLE
3p cloud storage provider Upload API bug fix

### PR DESCRIPTION
**Pull request by HarshJain27**
This PR has following changes and corresponding test code update

1. Changed datatype of field **lastModified** from Date to number : This is required because of type incompatibility between the calling app API and SDK API, and  because of which the integration fails.
2. Added a new field named **size** representing the file size in bytes, being congruent with the calling app API. 

Below PRs contain the original 3P API changes.
https://github.com/OfficeDev/microsoft-teams-library-js/pull/1152
https://github.com/OfficeDev/microsoft-teams-library-js/pull/1251

File uploads to 3P providers should work as expected after this change.

Unit Tests added:
No, Modified the unit tests as applicable.

Change file added:
Yes

### CODE REVIEW

File: change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Files upload API : 1. Changed datatype of `lastModified` field from Date to number 2. Added a new field `size` of type number",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json:3]: Please don't format this as a numbered list. This sentence shows up in public changelogs and the numbered list doesn't get formatted nicely. Try something like:

On the `File` interface changed the type of lastModified from `Date` to `number` and added a new field `size` that contains the size in bytes of the file.

File: change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "packageName": "@microsoft/teams-js",
+  "email": "jainharsh@microsof.com",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json:5]: missing the t in microsoft

File: change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json (hunk 1 above)
Comment (HarshJain27) [reply change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json]: Will follow this now onwards. Updated the file.

File: change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json (hunk 2 above)
Comment (HarshJain27) [reply change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json]: Corrected. Thanks for pointing out.

File: packages/teams-js/src/private/files.ts [hunk 3]
```diff
@@ -415,23 +415,28 @@ export namespace files {
...
     name: string;
     /**
-     * The file path to uniquely identify it within the file hierarchy
+     * A string containing the path of the file relative to the ancestor directory the user selected
      */
     webkitRelativePath?: string;
+    /**
+     * The size, in bytes, of the data contained in the Blob object
+     */
+    size: number;
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/private/files.ts:439]: Size is already a member of the Blob interface, why do you need to override it with the same thing here?

**Review by maglims — APPROVED** [review]

**Pull request merged**